packages/salesforcedx-vscode-soql/src/channel/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { ChannelService } from '@salesforce/salesforcedx-utils-vscode/out/src/commands';
+
+export const channelService = ChannelService.getInstance(
+  // i18n
+  'SOQL Extension'
```

Comment:
Update this to use a label that can be localized

Reply:
@lcampos See the `feat(soql): nls-enable channel name` commit.

Reply:
Looks good 👍